site/gdocs/components/centered-article.scss

```diff
@@ -807,33 +807,42 @@ p.article-block__text + #article-licence {
     }
 }
 
-.article-block__aside-right figcaption {
-    border-left: 1px solid $blue-20;
-    padding-left: 24px;
-    a {
-        @include owid-link-90;
+figure[class*="article-block__aside"] {
+    position: relative;
+    margin: 0;
+
+    figcaption {
+        margin-bottom: 8px;
+        a {
+            @include owid-link-90;
+        }
+
+        @include md-up {
+            position: absolute;
```

Comment:
This is a trick to get the text to escape its grid container and ignore the grid rows. 

It will break if the aside is very long and near the bottom of the article (by overflowing into the footer) or comes immediately before a full-width section, but the upside is that it will never create a large space before the next component 

e.g. 
![image](https://github.com/user-attachments/assets/cec72c95-c519-40be-9e21-97225e3f5864)


The alternative is that we set `grid-row: auto / span 3` on the aside block and hope that spanning 3 rows is always sufficient. This is possibly still a better solution, I'm on the fence.